core/src/apps/ethereum/tokens.py

```diff
@@ -13,10 +14,10 @@ def __init__(self, symbol: str, decimals: int) -> None:
 
 
 def token_by_chain_address(chain_id: int, address: bytes) -> TokenInfo:
-    if False:
+    if TYPE_CHECKING:
```

Comment:
This is wrong.

We use `if False` in this function for something else than typing.

Reply:
Thanks for pointing it out. Reverted in 1afb0c8